src/policy/immix/immixspace.rs

```diff
@@ -547,9 +605,12 @@ impl<VM: VMBinding> ImmixSpace<VM> {
 
     /// Check if an object is pinned.
     #[inline(always)]
-    fn is_pinned(_object: ObjectReference) -> bool {
-        // TODO(wenyuzhao): Object pinning not supported yet.
-        false
+    fn is_pinned(object: ObjectReference) -> bool {
+        use crate::mmtk::SFT_MAP;
+        use crate::policy::sft_map::SFTMap;
+        SFT_MAP
+            .get_checked(object.to_address())
+            .is_object_pinned(object)
```

Comment:
This is a private method, so it is not called from elsewhere.  I think we can bypass the SFT lookup (or `debug_assert!` that the input `object` is in the current ImmixSpace), and call `self.is_object_pinned`, instead.

Oh there is no `self`! Maybe we can remove this method and call `self.is_object_pinned` instead.

Reply:
Fixed.